pkg/webhook/resources/virtualmachine/mutator_test.go

```diff
@@ -136,23 +136,6 @@ func TestPatchResourceOvercommit(t *testing.T) {
 			},
 			setting: `{"cpu":1000,"memory":1000,"storage":800}`,
 		},
-		{
-			name: "replace old guest memory",
```

Comment:
Should this test case be reverted and some new test cases upon hotplug mode be added?